packages/block-editor/src/components/block-styles/index.js

```diff
@@ -60,7 +60,9 @@ function BlockStyles( {
 		const scrollContainer = document.querySelector(
 			'.interface-interface-skeleton__content'
 		);
-		setContainerScrollTop( scrollContainer.scrollTop + 16 );
+		const scrollTop = scrollContainer?.scrollTop || 0;
+		// Add the hardcoded equivalent of the container's right position.
+		setContainerScrollTop( scrollTop + 16 );
```

Comment:
Ignorable comment: in the contexts where `scrollContainer` is null, do we still need the 16?
 
```suggestion
		if( scrollContainer ) {
		// Add the hardcoded equivalent of the container's right position.
		setContainerScrollTop(  scrollContainer.scrollTop + 16 );
		}
```

Reply:
Thanks for 👀 @alshakero !

> : in the contexts where scrollContainer is null, do we still need the 16

Not really, but I don't see any problem with leaving it at `16`. Normally, `16` (the same as the `right` value in the CSS) is the minimum top position value we need to display the block preview. 

If the `scrollContainer` isn't there, we can assume the block preview won't be rendered at all since the Slot component lives within it. 

That's the theory 😆